src/pages/home/ReportScreen.tsx

```diff
@@ -643,6 +643,13 @@ function ReportScreen({
         });
     }, [isInaccessibleWhisper]);
 
+    useEffect(() => {
+        if (!!report.lastReadTime || !ReportUtils.isTaskReport(report)) {
```

Comment:
why did we add `!ReportUtils.isTaskReport(report)`? and are you sure we're fixing the root cause?

Reply:
@suneox it might fix similar issues for different kinds of reports wdyt?

Reply:
Lets add a comment about the fix